### ISSUE TITLE
Error: Cannot find package "mongo-id"

### ISSUE THREAD

**Issue by eminx** [adtribute/pub-sub-lite#7]
I'm trying to install and run my existing app locally with this package.

However I bump into some issues:

First I wasn't able to install it because I had to update my app. Fair enough I did the update and it installed.
Second I had the pool size issue that asked me to reorder pub-sub-lite at .packages file due to setting mongo configuration. I did that too.
Then I asked me to install the mongo-id package, which I did. Then it's still not recognising that package is installed - still asking for it. 

Got stuck here so far. Any suggestions?

Thinking that the above points would be good to add in the readme doc, since it's not that straightforward as it is currently explained.

Thanks!

**Comment by eminx** [comment]
In addition, now I pushed a bit the limits and moved the mongo-id package above this package (in contrast to the suggestion), then it seemed to cover the mongo-id problem. 

But then I bumped into another issue asking me to install ddp-client. I did. and same again and then now I'm giving up.

I was hopeful to do some performance optimisation with almost no code change but now I find myself spent a few hours without getting anywhere with this. Am I doing something wrong or??
